website/website/pages/en/index.js

```diff
@@ -92,117 +28,60 @@ class Index extends React.Component {
       </Container>
     );
 
-    const FeatureCallout = () => (
-      <div
-        className="productShowcaseSection paddingBottom"
-        style={{textAlign: 'center'}}>
-        <h2>Feature Callout</h2>
-        <MarkdownBlock>These are features of this project</MarkdownBlock>
-      </div>
-    );
-
-    const TryOut = () => (
-      <Block id="try">
+    const Features = () => (
+        <div>
+      <Block layout="threeColumn">
+          {[
+              {
+                  content: 'A server written in Java which allow to send push notifications to different mobile operating systems and web application',
+                  image: `${baseUrl}img/notification.svg`,
```

Comment:
possible to make the image also a link? This is what I clicked first. If its a pain, would making them into cards be a better option?

Reply:
Done!